BossMod/Modules/Endwalker/HuntS/Burfurlur.cs

```diff
@@ -11,66 +11,61 @@ public enum AID : uint
     {
         AutoAttack = 872, // Boss->player, no cast, single-target
         Trolling = 27316, // Boss->self, no cast, range 10 circle
-        QuintupleInhale1 = 27307, // Boss->self, 4.0s cast, range 27 ?-degree cone
-        QuintupleInhale24 = 27308, // Boss->self, 0.5s cast, range 27 ?-degree cone
-        QuintupleSneeze1 = 27309, // Boss->self, 5.0s cast, range 40 ?-degree cone
-        QuintupleSneeze24 = 27310, // Boss->self, 0.5s cast, range 40 ?-degree cone
-        QuintupleInhale35 = 27692, // Boss->self, 0.5s cast, range 27 ?-degree cone
-        QuintupleSneeze35 = 27693, // Boss->self, 0.5s cast, range 40 ?-degree cone
+        QuintupleInhale1 = 27307, // Boss->self, 4.0s cast, range 27 45-degree cone
+        QuintupleInhale24 = 27308, // Boss->self, 0.5s cast, range 27 45-degree cone
+        QuintupleSneeze1 = 27309, // Boss->self, 5.0s cast, range 40 45-degree cone
+        QuintupleSneeze24 = 27310, // Boss->self, 0.5s cast, range 40 45-degree cone
+        QuintupleInhale35 = 27692, // Boss->self, 0.5s cast, range 27 45-degree cone
+        QuintupleSneeze35 = 27693, // Boss->self, 0.5s cast, range 40 45-degree cone
         Uppercut = 27314, // Boss->self, 3.0s cast, range 15 120-degree cone
         RottenSpores = 27313, // Boss->location, 3.0s cast, range 6 circle
     };
 
-    class QuintupleSneeze : Components.GenericAOEs
+    class QuintupleSneeze : Components.GenericRotatingAOE
```

Comment:
It's only barely a 'rotation' component (the sequence has very arbitrary increments), don't making it a rotation actually simplifies anything...

Reply:
I don't know if it is now more complicated than before, but at least the activation times seem to be perfect. And the colors are now consistant with other mechanics now where sequences exist